geode-protobuf/src/main/java/org/apache/geode/internal/protocol/protobuf/security/SecureCacheImpl.java

```diff
@@ -52,7 +52,7 @@
   private final SecureFunctionService functionService;
 
   public SecureCacheImpl(InternalCache cache, Security security) {
-    this.cache = cache;
+    this.cache = cache.getCacheForProcessingClientRequests();
     this.security = security;
     this.functionService = new SecureFunctionServiceImpl(cache, security);
```

Comment:
this.cache? instead of cache....

Reply:
I'm sorry but I don't understand this comment.  The SecureCacheImpl needs to hold an InternalCacheForClientAccess, so this change seems necessary.

Reply:
The question was about passing the cache instance to SecureFunctionServiceImpl(); line#57 since its not changed in this pr, the gui was displaying it in extreme left...

Reply:
I see - thanks, I'll fix that